automation/test.sh

```diff
@@ -409,9 +409,8 @@ if [ -z "$KUBEVIRT_QUARANTINE" ]; then
     else
         export KUBEVIRT_E2E_SKIP="QUARANTINE"
     fi
-    # quarantine test_id:3145 only for nonroot lanes
```

Comment:
@xpivarc Hi. I think you can remove the whole condition block:
```
    if [[ $KUBEVIRT_NONROOT =~ true ]]; then
        export KUBEVIRT_E2E_SKIP="${KUBEVIRT_E2E_SKIP}"
    fi
```

Reply:
@enp0s3 done.